src/index.css

```diff
@@ -2,36 +2,179 @@
 @import url('https://fonts.googleapis.com/icon?family=Material+Icons');
 
```

Comment:
CSS modules could have been leveraged more in this project to avoid declaring so many global styles that are only used in a single component